Schedule.API/Controllers/ExaminationController.cs

```diff
@@ -90,5 +96,19 @@ public IActionResult AdvancedSearch(ExaminationAdvancedFilterDto dto)
             dto.PatientId = Int32.Parse(HttpIdentityHandler.GetUserIdFromRequest(HttpContext.Request));
             return Ok(_examinationService.Search(dto));
         }
+
+        [HttpPost]
+        [Route("check-rooms")]
+        public IActionResult GetUnavailableRooms(EquipmentRelocationDto dto)
+        {
+            return Ok(_equipmentRelocationSchedulingService.GetUnavailableRooms(dto));
+        }
+
+        [HttpPost]
+        [Route("get-doctors")]
+        public IActionResult GetDoctorsByRoomsAndShifts(EquipmentRelocationDto dto)
+        {
+            return Ok(_equipmentRelocationSchedulingService.GetDoctorsByRoomsAndShifts(dto));
+        }
```

Comment:
The Route is not suitable for this method. When you set route "get-doctors", I would expect to get all doctors, not only few that have connection with rooms and shifts.

Reply:
I renamed it to get-doctors-by-rooms-and-shifts.